src/styles/blocks/header.scss

```diff
@@ -33,10 +33,15 @@
   }
 
   &__link {
+    transition: $transition-duration;
```

Comment:
Please specify to which properties you want to apply the transition, as you did in catalog.scss
For now it applies to all the properties and it's not good for performance
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x